docs/run/nodes.md

```diff
@@ -1,48 +1,45 @@
 ```
 
-
-## **Start the daemon**
-
-Next, start the Ceramic daemon.
+### 2. Start the Ceramic node
+This starts a Ceramic node with a standard configuration on the [Clay Testnet](../learn/networks.md#clay-testnet) at `https://localhost:7007`. Behind the scenes a properly configured [IPFS](../learn/glossary.md#ipfs) instance is started, then IPFS is used to create an instance of the Ceramic [JS Core Client](../clients/javascript) connected to the [Clay Testnet](../learn/networks.md#clay-testnet). Once Ceramic Core is running, an HTTP server is started that serves the Ceramic [HTTP API](../reference/http-api.md).
 
 ```bash
 $ ceramic daemon
 ```
 
-This command starts a Ceramic node with a standard cofiguration. By default the daemon runs a bundled IPFS instance. First, a properly configured IPFS instance is started. Then, IPFS is used to create an instance of Ceramic core. Once Ceramic core is running, an HTTP server is started that serves the Ceramic HTTP API. The node is now ready to accept requests from the local environment or remotely.
-
-### Using an external IPFS node
-
-It's also possible to run a separate service for IPFS using the `@ceramicnetwork/ipfs-daemon` npm package. 
+### 3. Use an external IPFS node
+By default the JS CLI starts a node with a bundled IPFS instance. If you would like to instead run a separate service for IPFS, use the `@ceramicnetwork/ipfs-daemon` npm package. 
 
+### 4. Configure a network
+By default the JS CLI starts a node on the Clay Testnet. If you would like to use a different network, you can specify this using the `--network` option. Currently, it is only possible to host nodes on the Clay Testnet. We will add the ability to run nodes on [Mainnet](../learn/networks.md#mainnet) in the near future.
 
-## **Network connectivity**
+### 5. Expose endpoints
+If you want your node and the streams created on it to be discoverable to the rest of the network, make sure the appropriate ports used by IPFS and the Ceramic HTTP API (by default this is port 7007) are publicly accessible. For example, if you are behind a router you may need to set up port forwarding. 
 
-### Expose ports
+### 6. Enable peer discovery
+To enable peer discovery, add your node to the appropriate Ceramic [`peerlist`](https://github.com/ceramicnetwork/peerlist) file by submitting a pull request. This is used as a temporary measure to streamline peer discovery until `js-ipfs` supports DHT lookups.
 
-If you want your node to be discoverable to the rest of the Ceramic Network, make sure the appropriate ports used by IPFS and the Ceramic HTTP API (by default this is port 7007) are publicly accessible. For example, if you are behind a router you may need to set up port forwarding. 
+### 7. Gateway mode
+To setup your node with read-only capabilities, enable gateway mode. Users of this node will only be able to read data from the network, but writes will be disabled.
 
-### Enable peer discovery
+## **Next steps**

```

Comment:
Feels like this page needs some information about data persistence. Something on the S3 state store and various IPFS pinning backends

Reply:
To be done by Joel/Val when they improve this page. Agreed, overall this page is somewhat incomplete.